pkgs/development/python-modules/ansible/default.nix

```diff
@@ -53,6 +54,7 @@ buildPythonPackage {
     ttp
     xmltodict
     # ansible.windows
+    jmespath
```

Comment:
So not part of an ansible collection? Because it surely doesn't fit directly below `# ansible.windows`.